src/lib_c/x86_64-windows-msvc/c/winapi.cr

```diff
@@ -0,0 +1,241 @@
+lib LibC
+  STD_INPUT_HANDLE  = 0xFFFFFFF6_u32
+  STD_OUTPUT_HANDLE = 0xFFFFFFF5_u32
+  STD_ERROR_HANDLE  = 0xFFFFFFF4_u32
+
+  FILE_TYPE_UNKNOWN = 0x0000
+  FILE_TYPE_DISK    = 0x0001
+  FILE_TYPE_CHAR    = 0x0002
+  FILE_TYPE_PIPE    = 0x0003
+  FILE_TYPE_REMOTE  = 0x8000
+
+  GENERIC_EXECUTE = 0x20000000
+  GENERIC_WRITE   = 0x40000000
+  GENERIC_READ    = 0x80000000
+
+  CREATE_NEW        = 1
+  CREATE_ALWAYS     = 2
+  OPEN_EXISTING     = 3
+  OPEN_ALWAYS       = 4
+  TRUNCATE_EXISTING = 5
+
+  FILE_FLAG_OVERLAPPED = 0x40000000
+
+  # source https://msdn.microsoft.com/en-us/library/windows/desktop/aa383751(v=vs.85).aspx
+  alias Long = Int32
+  alias Word = UInt16
+  alias WChar = UInt16
+  alias DWord = UInt32
+  alias Handle = Void*
+  alias SizeT = UInt64
+  alias BOOL = Int32 # FIXME maybe it need to be removed because it can be confused with Bool
+
+  INVALID_HANDLE_VALUE = Pointer(Void).new((-1).to_u64)
+  INFINITY             = 0xFFFFFFFF_u32
+
+  fun _DuplicateHandle = DuplicateHandle(source_process : Handle, source : Handle, target_process : Handle, target : Handle*, desired_access : DWord, inherit_handle : Bool, options : DWord) : Bool
+
+  # member names are not original otherwise they would be really confusing
+  struct OVERLAPPED
+    status : SizeT
+    bytes_transfered : SizeT
+    offset : UInt64
+    event : Handle
+  end
+
+  struct SECURITY_ATTRIBUTES
+    nLength : DWord
+    lpSecurityDescriptor : Void*
+    bInheritHandle : BOOL
+  end
+
+  fun _GetStdHandle = GetStdHandle(std_handle : DWord) : Handle
```

Comment:
Because a `fun` must follow the same rules than `def` even under a `lib` definition. I'd love if this rule was relaxed, only for `fun` under `lib` definitions —it's quite common for C symbols to be "namespaced" with an uppercased prefix: `LLVM`, `SDL2_`, ... or to have camelcased functions as per the Win32 API.